src/app/pages/contact/contact.component.ts

```
@@ -0,0 +1,129 @@
+    } catch (error) {
+      this.toastService.show({ title: 'Failed to fetch PoW challenge. Please try again later!', type: 'error' });
+      throw error;
+    }
+  }
+
+  solvePoW(challenge: string): void {
+    let solution = 0;
+    // difficulty set to 4
+    const target = '0'.repeat(4);
+    const startTime = Date.now();
+    while (true) {
```

ESLint

[error] Unexpected constant condition.

(no-constant-condition)